### PR TITLE
Release v2.0.1 in develop

### PR DESCRIPTION
**Pull request by giovannipizzi**

### CODE REVIEW

**Comment by giovannipizzi** [comment]
I needed to do a merge because in develop we only support py>=3.6.

File: aiida_wannier90/workflows/minimal.py [hunk 1]
```diff
@@ -262,7 +262,6 @@ def run_w90_pp(self):
 
     def run_pw2wan(self):
         """Run pw2wannier90.x."""
-        self.out('nscf_output', self.ctx.pw_nscf.outputs.output_parameters)
```
Comment (greschd) [inline aiida_wannier90/workflows/minimal.py]: Why was the `nscf_output` removed?

Comment (qiaojunfeng) [reply aiida_wannier90/workflows/minimal.py]: Just because it is repeated, details here https://github.com/aiidateam/aiida-wannier90/pull/96

Comment (greschd) [reply aiida_wannier90/workflows/minimal.py]: Ah, thanks! I will merge this, then.

**Pull request merged**